docs/reference/configuration/Single Sign-On/recoving-lost-credentials.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Recovering lost Keycloak credentials
+---
+
+This procedure describes how to recover lost Keycloak credentials for UDS Core. It leverages the [Admin bootstrap and recovery](https://www.keycloak.org/server/bootstrap-admin-recovery) feature of Keycloak.
+
+:::caution
+This procedure requires at least 1.5G of memory allocated to the Keycloak container. You may need to temporarily increase the memory limit before starting the recovery process. If the `JAVA_OPTS_KC_HEAP` environment variable is used, ensure the -XX:MaxRAM setting corresponds to the container memory limits.
+:::
+
+The procedure involves creating a new user with administrator privileges, logging into that user, and recovering the lost credentials. First, create a new temporary admin user called `temp-admin` with the password `temp-admin`:
```

Comment:
nit: even if temporary I don't think we should be encouraging users to make weak credentials like this, since it could be used during this brief period. Probably better to leave this for the end user to set a memorable, but secure, password?

Reply:
You're absolutely right. Let me reword this piece.